x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -671,6 +694,53 @@ export function dataFrameAnalyticsRoutes({ router, mlLicense, routeGuard }: Rout
     })
   );
 
+  /**
+   * @apiGroup DataFrameAnalytics
+   *
+   * @api {get} api/data_frame/analytics/fields/:indexPattern Get index pattern fields for analytics
+   * @apiName GetDataFrameAnalyticsIndexFields
```

Comment:
This needs to be added to the list of routes in `routes/apidoc.json`

Reply:
Added in fa3e81bdcbfd5b0fe270287619220c7907a36a59